src/minimal-webapi-openidconnect-v2/MinimalWebApi.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Identity.Web" Version="1.*" />
+    <PackageReference Include="Swashbuckle.AspNetCore" Version="6.1.5" />

```

Comment:
no swagger even more minimalist is fine

Reply:
done | addressed from [82ae1](https://github.com/Azure-Samples/ms-identity-docs-code-csharp/pull/2/commits/82ae1648a598ce15508ffd53dd7998881c0eac45)

Reply:
🪓